app/src/main/java/a/a/a/Jx.java

```diff
@@ -299,20 +315,21 @@ public String a() {
             }
             sb.append(a);
         }
-        if (!this.f.a(this.c.getActivityName()).a() || this.c.fileName.contains("_fragment")) {
-            sb.append("initializeLogic();");
-            sb.append(a);
-        } else {
+        sb.append(a);
+        if (!this.c.fileName.contains("_fragment")) {
             jq jqVar = this.f;
-            sb.append(Lx.a(jqVar.g, jqVar.a(this.c.getActivityName()).c));
+            //sb.append(Lx.a(jqVar.g, jqVar.a(this.c.getActivityName()).c));
```

Comment:
Just a note: You don't have to comment out lines, git will automatically save what on that line was.

Reply:
Same, I forgot to remove